src/main/webapp/script.js

```diff
@@ -36,4 +36,124 @@ function displayError() {
   // TODO: Write error to user interface
   console.log('Geolocation not enabled');
   alert(alertMessage);
+}
+
+//Array of the (currently 6 for this demo build) 15 listings gathered from the fetch request
+const listingsArray = [];
```

Comment:
Should this be a 'let' since elements will be added to the array?

Reply:
Yes, good catch